tests/test_utils.py

```diff
@@ -6,10 +6,18 @@
 #
 ##############################################################################
 
+# mypy: disable-error-code="misc"
```

Comment:
Since it's needed at only two places, I recommend locally ignoring it instead of here.